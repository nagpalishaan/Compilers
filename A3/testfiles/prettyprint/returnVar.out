int n() {
int a = 5;
return a;
}

