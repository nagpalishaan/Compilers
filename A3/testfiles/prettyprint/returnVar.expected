Program
  List
    Function
      IdDecl ID="n"
      Opt
      Opt
        List
          DeclareVar
            IdDecl ID="a"
            Opt
              Numeral NUMERAL="5"
          Return
            IdUse ID="a"
